fractions/microprofile/microprofile-fault-tolerance/README.adoc

```diff
@@ -1,5 +1,5 @@
 [id='microprofile-fault-tolerance']
-= MicroProfile Fault Tolerance
+= Eclipse MicroProfile Fault Tolerance
```

Comment:
I think we shouldn't change the headings, as they correspond to fraction names. The 1st sentence below already uses the full name.

Reply:
Makes sense, I'll revert the changes to the titles.